src/constants/routes.ts

```diff
@@ -31,18 +31,20 @@ export enum profileRoutes {
   governance = "governance",
 }
 
-export enum adminRoutes {
+export const adminRoutes = {
```

Comment:
By turning `adminRoutes` into a constant, it is possible to assign same values to different fields, in this case `index` and `idnex_review` have the same value `""`.
This is useful in general, but in this case it was necessary so that it becomes possible to assign different page titles to the same index path `""`.